docs/root/configuration/http/http_filters/lua_filter.rst

```diff
@@ -390,16 +390,34 @@ httpCall()
 
 .. code-block:: lua
 
-  local headers, body = handle:httpCall(cluster, headers, body, timeout, asynchronous)
+  local headers, body = handle:httpCall(cluster, headers, body, timeout, request_options)
 
 Makes an HTTP call to an upstream host. *cluster* is a string which maps to a configured cluster manager cluster. *headers*
 is a table of key/value pairs to send (the value can be a string or table of strings). Note that
 the *:method*, *:path*, and *:authority* headers must be set. *body* is an optional string of body
 data to send. *timeout* is an integer that specifies the call timeout in milliseconds.
 
-*asynchronous* is a boolean flag. If asynchronous is set to true, Envoy will make the HTTP request and continue,
-regardless of the response success or failure. If this is set to false, or not set, Envoy will suspend executing the script
-until the call completes or has an error.
+*request_options* controls some of the HTTP call options. This option can be omitted, supplied with
+a boolean flag, or a table.
+
+- If *request_options* is omitted or set with false, Envoy will suspend executing the script until
+  the call completes or has an error.
+- If *request_options* is a boolean and set with true, Envoy will make the HTTP request and
+  continue, regardless of the response success or failure.
+- If *request_options* is a table, in addition to the asynchronicity setting, the tracing sampling
+  flag can also be specified. Currently, the supported keys are "asynchronous" and "sampled". Some
+  examples are shown below:
+
+  .. code-block:: lua
+
+    -- Create a fire-and-forget HTTP call.
+    local request_options = {["asynchronous"] = true}
```

Comment:
I think the result of `{asynchronous = true}` vs. `{["asynchronous"] = true}` will be compiled as the same code (both will allocate memory for holding the "asynchronous" string as key). The latter is "generic" way of doing it since it can accommodate keys that are not legal identifiers. But yeah, I can update it to `{asynchronous = true}`

Reply:
I decided to keep this as is, since we consistently use `""` to surround keys, including the scripts in the tests.